ddl/index.go

```diff
@@ -544,7 +544,7 @@ func (w *worker) onCreateIndex(d *ddlCtx, t *meta.Meta, job *model.Job, isPK boo
 				func() {
 					addIndexErr = errCancelledDDLJob.GenWithStack("add table `%v` index `%v` panic", tblInfo.Name, indexInfo.Name)
 				}, false)
-			return w.addTableIndex(tbl, indexInfo, reorgInfo)
```

Comment:
Whether a index is using global index or not is decided by the index info itself, so we do not need to add an extra parameter, instead, indexInfo could be modified.